apps/condo/domains/meter/schema/Meter.js

```diff
@@ -185,6 +186,22 @@ const Meter = new GQLListSchema('Meter', {
                 return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown \`dv\``)
             }
         },
+        afterChange: async ({ context, operation, originalInput, updatedItem }) => {
+            if (operation === 'update') {
+                const deletedMeterAt = get(originalInput, 'deletedAt')
+
+                if (deletedMeterAt) {
+                    const meterId = get(updatedItem, 'id')
+                    const meterReadings = await MeterReading.getAll(context, { meter: { id: meterId } })
+
+                    for (const reading of meterReadings) {
+                        await MeterReading.update(context, reading.id, {
```

Comment:
probably it's ok for now, but not for future. Add TODO to move this to the worker